app/views/comment/_single_comment.html.erb

```diff
@@ -25,10 +25,12 @@
     <p class="event_actions">
       <% if comment.id %>
         <% if @user && @user.admin_page_links? %>
-          <%= link_to "Admin", edit_admin_comment_path(comment) %>
+          <%= link_to "Admin", edit_admin_comment_path(comment) %> |
         <% end %>
 
-        <!-- | <%= link_to _('Report abuse'), comment_path(comment) %> -->
+        <%= link_to _('Report abuse'),
```

Comment:
I just literally copied the link to this button code, duped it and pasted it next to it with "report" as the text. Will probably need to make the classes applied to it a bit more generic.